numpy/core/tests/test_umath.py

```diff
@@ -686,6 +686,58 @@ def test_float_remainder_corner_cases(self):
                 assert_(np.isnan(fmod), 'dt: %s, fmod: %s' % (dt, rem))
 
 
+class TestDivisionOverflows:
+    overflow_results = {
+        np.remainder: '0',
+        np.fmod: '0',
+        np.floor_divide: '-np.iinfo(dividend_dtype).min',
+        np.divmod: '(-np.iinfo(dividend_dtype).min, 0)'
+    }
+
+    @pytest.mark.parametrize("dividend_dtype",
+            np.sctypes['int'])
+    @pytest.mark.parametrize("divisor_dtype",
+            np.sctypes['int'])
+    @pytest.mark.parametrize("operation",
+            [np.remainder, np.fmod, np.divmod, np.floor_divide])
+    def test_overflows_no_raises(self, dividend_dtype, divisor_dtype, operation):
+        arrays = [np.array(
+                    [dividend_dtype(
+                        np.iinfo(dividend_dtype).min)]*i
+                    ) for i in range(1, 129)]
+        divisor = np.array(-1, dtype=divisor_dtype)
```

Comment:
```suggestion
        divisor = np.array([-1], dtype=divisor_dtype)
```

Reply:
Wow! I wasted more time than I'm proud to admit! Thanks!